orders/src/main/java/com/prumoitstore/order/domain/OrderDetail.java

```diff
@@ -0,0 +1,38 @@
+package com.prumoitstore.order.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Builder
+@Table(name = "orderDetails")
+public class OrderDetail {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "order_detail_id")
+    private Integer id;
+    private Date createdAt;
+    private  Date updateAt;
+    private Date deletedAt;
+    private Double orderTotal;
+    @Enumerated(EnumType.STRING)
+    private OrderStatus orderStatus;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "fk_client_id")
```

Comment:
remove "fk_" word in the name of the column. This is not a pattern from the database naming.